pages/advanced-algorithms/available-algorithms/refactor.mdx

```diff
@@ -510,6 +510,192 @@ CALL refactor.invert(l) YIELD input, output RETURN input, output;
 
 ![](/pages/advanced-algorithms/available-algorithms/refactor/rel_after.png)
 
+### `normalize_as_boolean()`
+
+Normalizes properties to true or false depending whether the property value is contained in `true_values` or `false_values`. If the property value is not contained in either list the property is removed. Throws an exception if both lists contain the same value.
+
+{<h4> Input: </h4>}
+
+- `entity: Node|Relationship` ➡  Node or relationship whose property value will be normalized.
+- `property_key: string` ➡  Key of the property to be normalized.
+- `true_values: List[Any]` ➡  List of property values which will be normalized to true.
+- `false_values: List[Any]` ➡  List of property values which will be normalized to false.
+
+{<h4> Usage: </h4>}
+
+Create a graph using the queries below:
+
+```cypher
+CREATE (n1:Node1 {property: "YES"}) CREATE (n2:Node2 {property: "NO"}) CREATE (n3:Node3 {property: "MAYBE", exists:true}) CREATE (n4:Node4) CREATE (n5:Node5 {exists:false}) CREATE (n1)-[:Rel1 {property:"YES"}]->(n2) CREATE (n2)-[:Rel2 {property:"NO"}]->(n3) CREATE (n3)-[:Rel3]->(n4) CREATE (n4)-[:Rel4 {property:"MAYBE", exists:"YES"}]->(n5);
+```
+
+The following query will change every property called `property` to true if it is equal to `YES` and false if it is equal to `NO`, otherwise it will delete the property:
```

Comment:
```suggestion
The following query will set  every property called `property` to `true` if it is equal to `YES`, and to `false` if it is equal to `NO`, otherwise it will delete the property:
```